libraries/AP_Motors/examples/AP_Motors_test/AP_Motors_test.cpp

```diff
@@ -120,6 +120,15 @@ void setup()
                     exit(1);
                 }
 
+            } else if (strcmp(cmd,"dual_mode") == 0) {
+                if (frame_class != AP_Motors::MOTOR_FRAME_HELI_DUAL) {
+                    ::printf("dual_mode only supported by dual heli frame type (%i), got %i\n", AP_Motors::MOTOR_FRAME_HELI_DUAL, frame_class);
+                    exit(1);
+                }
+
+                // look away now, more dodgy param access.
+                AP_Int8 *dual_mode = (AP_Int8*)motors + AP_MotorsHeli_Dual::var_info[1].offset;
```

Comment:
How much work would it be to add a proper param accessor in AP_Motors_test?  Just thinking to make it easier to add other configs in the future?